content/klc/G2.1.adoc

```diff
@@ -1,21 +1,29 @@
 +++
-brief = "Defining generic and atomic parts"
+brief = "Defining generic and fully specified symbols"
 +++
 
-In general terms, the _symbols_ in the KiCad library can be categorized into two groups:
+KiCad generally has two workflows with regards to assign a footprint to a symbol. It can be done in the library or using the "assign footprints tool" at the end of the design process.
 
 **Generic symbols**
 
-Generic symbols can be used with multiple footprints, and _do not_ have a default footprint assigned. Generic symbols allow flexibility in the design workflow. Symbols are first added to the schematic, and the footprint is selected before PCB layout. Using generic symbols allows a small number of library symbol elements to represent a _very large number_ of possible component combinations.
+Generic symbols can be used with multiple footprints, and _do not_ have a default footprint assigned. Generic symbols allow flexibility in the design workflow. For this workflow the symbol placed into the schematic is typically selected to fit the intended function. The footprint assignment happens later when the exact part number is selected. (At the latest when starting with the layout) Using generic symbols allows a small number of library symbol elements to represent a _very large number_ of possible component combinations.
```

Comment:
Missing a period for the sentence in quotes. And maybe that sentence isn't even needed?

Reply:
Do you mean the sentence in parenthesis?

> (At the latest when starting with the layout)?

I think that it would be proper for the period before to be moved after it, so:

> The footprint assignment happens later when the exact part number is selected (At the latest when starting with the layout).

But I also wonder if the part in parenthesis is really needed, leaving it out might be better?

Reply:
removed that part and reworded section